packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -688,7 +688,12 @@ abstract class Element implements BuildContext {
   Widget get widget => _widget;
   Widget _widget;
 
-  bool _active = false;
+  RenderObjectOwner _owner;
+  RenderObjectOwner get owner => _owner;
+  void set owner(RenderObjectOwner value) {
```

Comment:
Having a public setter for this value is kind of scary.  IMHO, we should have a separate notion of ownership for Elements than for RenderObjects.  Instead of boostrapping the Element owners from RenderObjectToWidgetElement, we'd have an explicit bootstrapping as part of runApp.